package.json

```diff
@@ -210,7 +210,7 @@
     "ts-morph": "^9.1.0",
     "ts-node": "^10.6.0",
     "typescript": "^4.8.2",
-    "webpack": "^5.76.1",
+    "webpack": "~5.76.1",
```

Comment:
Is that intentional?

Reply:
It is to temporary fix an error with node 14 on CCv2